apache/superset
# fix: disappearing tooltips on dashboards

### SUMMARY
Pulls in the fix here: https://github.com/apache-superset/superset-ui/pull/1162

Tooltips on nvd3 charts are buggy on dashboards, because if you hover over a chart before the rest of them load, the tooltips that have already been hovered on before disappear. This was because the migration to emotion added an extra div in between the chart container and the nvd3 chart, so we couldn't properly manage that chart's tooltips.

This fixes the bug by searching for the chart container all the way up the dom to get the proper chart id.

### TESTING INSTRUCTIONS
Sync code to superset, load a dashboard with nvd3 charts, hover over a chart to see the tooltip, force refresh a different nvd3 chart, see the first tooltip again after the second chart reloads.

### ADDITIONAL INFORMATION



to: @ktmud @pkdotson @graceguo-supercat
